src/software/backend/simulation/physics_ball_test.cpp

```diff
@@ -15,8 +15,8 @@ TEST(PhysicsBallTest, test_get_ball_with_timestamp)
     auto physics_ball = PhysicsBall(world, ball_parameter);
     auto ball         = physics_ball.getBallWithTimestamp(Timestamp::fromSeconds(1.1));
 
-    EXPECT_TRUE(ball_parameter.position().isClose(ball.position(), 1e-7));
-    EXPECT_TRUE(ball_parameter.velocity().isClose(ball.velocity(), 1e-7));
+    EXPECT_TRUE(ball_parameter.position().isClose(ball.position(), 1e-4));
```

Comment:
What's the reason for the change to `1e-7`?

Reply:
Leftover from different ```isClose()``` definition. Changed back in latest update

Reply:
Leftover from different ```Point::isClose()``` definition. Changed back to ```1e-7``` in latest update